.kodiak.toml

```diff
@@ -2,4 +2,5 @@
 version = 1
 
 [merge]
-method = "merge"
\ No newline at end of file
+method = "merge"
+block_on_reviews_requested = true
```

Comment:
@Hemanthkolli I think the right way to do this is to use github branch protection https://kodiakhq.com/docs/config-reference#mergeblock_on_reviews_requested